app/services/claims/fee_calculator/calculate_price.rb

```diff
@@ -139,7 +139,7 @@ def fee_type_code_for(fee_type)
 
       def current_fee_types
         return @current_fee_types if @current_fee_types
-        ids = current_page_fees.map { |pf| pf[:fee_type_id] }
+        ids = current_page_fees.pluck(:fee_type_id)
```

Comment:
LGTM but somewhat confusing as it is using https://www.rubydoc.info/github/rails/rails/Enumerable:pluck rather than activerecord pluck (`current_page_fees` is not an activerecord object)

Reply:
I'm not clear exactly what `curent_page_fees` is but I wonder if there is a better way to do this. Something like:

```ruby
def current_fee_types
  @current_fee_types ||= current_page_fees.map(&:fee_type)
end
```

Reply:
current_page_fees is an array returned by an ajax call here if memory serves:

https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/261e13a847c6d99cdd30e8c37439ea38ffa92b8f/app/webpack/javascripts/modules/external_users/claims/fee_calculator/FeeCalculator.UnitPrice.js#L194-L201


I think we are getting into scope creep thought so would not do this now or in this PR

Reply:
> I think we are getting into scope creep thought so would not do this now or in this PR

Quite. I was thinking that it were an array of `Fee::BaseFeeType`s there could be a quick cleanup but it looks as though it is more involved.